repos/spack_repo/builtin/packages/sphexa/package.py

```diff
@@ -20,20 +20,33 @@ class Sphexa(CMakePackage, CudaPackage, ROCmPackage):
 
     license("MIT")
 
+    version("0.95", sha256="1007ffa97eb2085d50173676ec5e6387d1da7a8b78f204308223fbdbbecc60a1")
     version("0.93.1", sha256="95a93d0063ac8857b9be12c1aca24f5b2eef9dd4ffe8cf3f6b552a4dd54b940f")
     version("develop", branch="develop")
 
     variant("hdf5", default=True, description="Enable support for HDF5 I/O")
     variant("gpu_aware_mpi", default=True, description="GPU aware MPI")
 
+    variant(
+        "cxxstd",
+        default="20",
+        multi=False,
+        description="Compile using the specified C++ standard",
+        values=("20", "23"),
+    )
```

Comment:
the variant has been introduced, but if I'm not wrong you are not using it anywhere.